jetty-ee10/jetty-ee10-maven-plugin/src/main/java/org/eclipse/jetty/ee10/maven/plugin/Overlay.java

```diff
@@ -78,12 +80,13 @@ public String toString()
      * @param dir the directory into which to unpack the overlay
      * @throws IOException 
      */
-    public void unpackTo(File dir) throws IOException
+    public void unpackTo(File dir) throws IOException // TODO: change to Path
     {
         if (dir == null)
             throw new IllegalStateException("No overly unpack directory");
         //only unpack if the overlay is newer
-        if (!dir.exists() || (getResource().lastModified() > dir.lastModified()))
+        Instant dirLastModified = Files.getLastModifiedTime(dir.toPath()).toInstant();
+        if (!dir.exists() || (getResource().lastModified().isAfter(dirLastModified)))
```

Comment:
Extra parens non necessary. See how `lastModified()` is used without a null check, so better it does not return null.

Reply:
Done.